code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -215,6 +215,7 @@
 	M.adjustStaminaLoss(-2, 0)
 	M.Jitter(2)
 	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, rand(1,4))
+	M.adjust_bodytemperature(25 * TEMPERATURE_DAMAGE_COEFFICIENT, 0, BODYTEMP_NORMAL) //meth makes you warm
```

Comment:
```suggestion
	M.adjust_bodytemperature(2, 0, BODYTEMP_NORMAL) //meth makes you warm
```

Reply:
just to point out, 25 * TDC is the exact same as Coffee.